### DISCUSSION TITLE
Equipped item's ItemAttributes -> Entity attributes

### DISCUSSION THREAD

**Discussion by ShadowHunterCZE** [Minestom/Minestom#1089] [Help] [answered]
Hello,
does Minestom implement looping of ItemAttributes on equipped items and then place them on Entities? Or I'm doing something wrong?

For example I created a sword with MAX_HEALTH Attribute:
`ItemStack.builder(Material.WOODEN_SWORD).meta(meta -> meta.attributes(List.of(new ItemAttribute(UUID.randomUUID(), "hp",  Attribute.MAX_HEALTH, AttributeOperation.ADDITION, 10, AttributeSlot.MAINHAND)))).build()` 
The attributes are correctly on the item -> can get them from ItemStack#meta().getAttributes()

But:
1) The item doesn't show the attribute, it's automatically hidden?
2) When I equip it, it doesn't update player attributes or show more health on the hotbar.

Also another question:
Is it OK to create your own Attributes and use them (+- 30 attributes, def more later)? Or should I better use Tags and implement my own value scaling.

**Comment by MrlnHi** [comment, marked as answer]
No, you have to add the attributes to the entity yourself
